config/jupyterhub_config.py

```diff
@@ -13,12 +13,17 @@
 import nativeauthenticator
 
 from jupyterhub_config.custom_docker_spawner import CustomDockerSpawner
+from jupyterhub_config.custom_kube_spawner import CustomKubeSpawner
 from jupyterhub_config.kb_jupyterhub_auth import KBaseAuthenticator, kbase_origin
 
 c = get_config()
 
-# TODO remove USE_KBASE_AUTHENTICATOR and Jupyterhub native auth once the KBaseAuthenticator is fully implemented
-if os.environ.get('USE_KBASE_AUTHENTICATOR', 'false').lower() == 'true':
+def get_bool_env(key, default=False):
+    """Parse str boolean environment variables"""
+    value = os.environ.get(key, str(default)).lower()
+    return value in ('true', '1', 't')
+
+if get_bool_env('USE_KBASE_AUTHENTICATOR'):
     # Set the authenticator class to KBaseAuthenticator
     # ref: https://jupyterhub.readthedocs.io/en/latest/reference/authenticators.html#authenticators
     c.JupyterHub.authenticator_class = KBaseAuthenticator
```

Comment:
So this doesn't need to be removed any more? Seems like we're there so the native auth can be dumped...

Reply:
I thought we were. But we also have this ticket: https://kbase-jira.atlassian.net/browse/CDM-32 to keep native auth as fallback.

Reply:
The problem with switching between kbase auth and local auth is that if 2 different people have the same name in the different auth systems, the ownership of their files will switch back and forth, IIUC. That seems Really Bad.

Reply:
yea. I didn't create the ticket. Maybe we just want to leave a fallback option incase anything went wrong.

Reply:
I added a comment on the ticket: https://kbase-jira.atlassian.net/browse/CDM-32?focusedCommentId=98362

would you mind adding that link as a comment in the code? I wouldn't want future people to get the wrong impression that it's ok to switch back and forth

Reply:
👍